packages/@expo/prebuild-config/package.json

```diff
@@ -49,9 +49,6 @@
     "semver": "^7.6.0",
     "xml2js": "0.6.0"
   },
-  "peerDependencies": {
-    "expo-modules-autolinking": "2.0.0-preview.2"
-  },
```

Comment:
Afaik we never needed that peer dep. Prebuild config doesn't import `expo-modules-autolinking` in a classic way, it's trying to resolve it from the project root where we do expect the `expo` package to already be installed.